NitroxClient/Communication/Packets/Processors/CyclopsActivateHornProcessor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Packets;
+using System;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class CyclopsActivateHornProcessor : ClientPacketProcessor<CyclopsActivateHorn>
+    {
+        private PacketSender packetSender;
+
+        public CyclopsActivateHornProcessor(PacketSender packetSender) {
```

Comment:
Please match the style formatting done thoughout the application("Curly brace on a new line")

Reply:
Done in new push (https://github.com/Sunrunner37/Nitrox/pull/23/files)